packages/fe-container/src/components/help-icon/HelpIcon.tsx

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2021 Mia srl
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import React from 'react'
+
+import './HelpIcon.less'
+
+export const HelpIcon: React.FC = () => {
+  const clickHandler = () => {
```

Comment:
I suggest to extract this function outside the component, because it doesn't depend on the component rendering